pkg/burner/create.go

```diff
@@ -226,20 +229,23 @@ func (ex *Executor) replicaHandler(objectIndex int, obj object, ns string, itera
 			// verify objects can lead into a race condition when some objects
 			// hasn't been created yet
 			replicaWg.Add(1)
-			go func() {
-				createRequest(obj.gvr, ns, newObject, obj.Namespaced)
+			go func(n string) {
+				if obj.Namespaced {
+					n = ""
+				}
+				createRequest(obj.gvr, n, newObject)
```

Comment:
If the object is namespaced... why are you passing an empty string as namespace rather than ns?
The conditional should negate obj.Namespaced

```suggestion
			go func(n string) {
				if ! obj.Namespaced {
					n = ""
				}
				createRequest(obj.gvr, n, newObject)
```